backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -288,6 +296,92 @@ const establishRoutes = () => {
 	 *         description: removes the containers found in the request body from the user's favourites list
 	 */
 	router.delete('/favourites', hasAccessToTeamspace, deleteFavourites);
+
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers/{container}:
+	 *   patch:
+	 *     description: Updates the settings of a container
+	 *     tags: [Containers]
+	 *     operationId: updateSettings
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: ID of project
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *       - container:
+	 *         name: container
+	 *         description: ID of container
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     requestBody:
+     *       content:
+     *         application/json:
+     *           schema:
+     *             type: object
+     *             properties:
+	 *               name:
+	 *                 type: String
+	 *                 example: container1
+	 *               desc:
+	 *                 type: String
+	 *                 example: description1
+	 *               type:
+	 *                 type: String
+	 *                 example: type1
+	 *               surveyPoints:
+	 *                 type: array
+	 *                 items:
+	 *                   type: object
+	 *                   properties:
+	 *                     position:
+	 *                       type: array
+	 *                       items:
+	 *                         type: float
+	 *                         example: 23.45
+	 *                     latLong:
+	 *                       type: array
+	 *                       items:
+	 *                         type: float
+	 *                         example: 23.45
+	 *               angleFromNorth:
+	 *                 type: integer
+	 *                 example: 100
+	 *               elevation:
+	 *                 type: integer
+	 *                 example: 100
+	 *               unit:
+	 *                 type: string
+	 *                 example: mm
```

Comment:
use enum to specify the list of possible options

This has overlap with @Charence  stuff i think - might worth just seeing if he's done it properly and copy it 😆